Failure was the unrelated 
```
/home/buildnode/jenkins/workspace/swift-PR-Linux-smoke-test/branch-main/swiftpm/Tests/WorkspaceTests/RegistryPackageContainerTests.swift:421:39: error: type 'FingerprintCheckingMode' has no member 'none'
20:09:38             fingerprintCheckingMode: .none,
20:09:38                                      ~^~~~
```
from swiftpm that we fixed; rebuilding

--- a/stdlib/public/Concurrency/TaskStatus.cpp
+++ b/stdlib/public/Concurrency/TaskStatus.cpp
@@ -471,7 +471,6 @@ swift_task_detachChildImpl(ChildTaskStatusRecord *record) {
 SWIFT_CC(swift)
 static void swift_taskGroup_attachChildImpl(TaskGroup *group,
                                             AsyncTask *child) {
-  SWIFT_TASK_DEBUG_LOG("attach child task = %p to group = %p", child, group);
 
   // We are always called from the context of the parent
   //

